controllers/kafka/strimzi-connectors.go

```diff
@@ -237,9 +244,13 @@ func (c *StrimziConnectors) CheckIfConnectIsResponding() (bool, error) {
 
 		if res != nil && res.StatusCode == 200 {
 			body, err := ioutil.ReadAll(res.Body)
+			if err != nil {
+				return false, err
+			}
+
 			err = res.Body.Close()
 			if err != nil {
-				return false, nil
+				return false, err
```

Comment:
@fijshion, I stumbled upon this one, and don't we need to return the error here instead of `nil`?